doc/on-behalf-of.md

```diff
@@ -0,0 +1,50 @@
+# On-Behalf-Of Authorization
+
+The feature can be enabled with the checkbox from the plugin configuration screen.
+⚠️ When provisioning, you also need to set `oauthPassThru` to `true`, such as the following example has done.
```

Comment:
I feel like this section has too many exclamation marks. Can you remove this one?